ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -97,10 +98,12 @@ public void setRespiratoryField(String value) {
     }
 
     public void clearPatientBloodPressure1() {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(BLOOD_PRESSURE_FIELD_1));
```

Comment:
```suggestion
        waiter.until(ExpectedConditions.visibilityOfElementLocated(BLOOD_PRESSURE_FIELD_1));
```

Reply:
Why adding waiter calls ?  This call is made for elements which take long to be loaded/visible on the page and these fields do not need any waiter calls at all!

Reply:
> why adding waiter calls ? This call is made for elements which take long to be loaded/visible on the page and these fields do not need any waiter calls at all!

Oh thanks @kdaud , these calls are fixing the issue of visibility of an element brought by the ci, you need to confirm https://ci.openmrs.org/browse/REFAPP-UI-552/